### ISSUE TITLE
lxqt-globalkeys and Super key problems

### ISSUE THREAD

**Issue by wxl** [lxqt/lxqt#1605]
If one sets the Super key in lxqt-globalkeys to some action, then Openbox shortcuts using the Super key fail to work. Interestingly, shortcuts in lxqt-globalkeys using the Super key DO work. It just doesn't pass through to Openbox. 

##### Expected Behavior
 * Super key set to some action works
 * Super key in combination with some other key(s) in lxqt-globalkeys set to some action works
 * Super key in combination with some other key(s) set to some obviously Openbox-specific action (read: window manager action) works
 * If the Super key is set to some action, toggling it off will cause it to stop working
 * If the Super key is set to some action, and toggled off, then toggling it back on will cause it to start working
 * Language should be consistent (either Super or Meta is used)

##### Current Behavior
 * Super key set to some action works
 * Super key in combination with some other key(s) in lxqt-globalkeys set to some action works
 * Super key in combination with some other key(s) set to some obviously Openbox-specific action (read: window manager action) *dos not* work
 * Toggling the Super key shortcut off allows the Super key combos defined in lxqt-globalkeys and Openbox to work
 * After toggling the Super key shortcut back on, the previous behavior is not restored until the shortcut is deleted and added again
 * Language is inconsistent (Super and Meta are used in different contexts)

##### Steps to Reproduce (for bugs)
 1. Set Super key (shows as `Super_L` for US keyboards where two Super keys are rare) in lxqt-globalkeys to open the command `pcmanfm-qt`
 1. Set Super key + c (shows as `Meta+C`) in lxqt-globalkeys to open the command `lxqt-config`
 1. Ensure Openbox has a binding  for Super key + d that shows the desktop `<keybind key="W-d"><action name="ToggleShowDesktop"/></keybind>`
 1. Try all of the shortcuts above
 1. Toggle the Super key shortcut off in lxqt-globalkeys
 1. Try the Super key combos above
 1. Toggle it back on
 1. Try all the shortcuts above

##### Context
It has been [mentioned][1] [here][2] [many][3] [times][4] that folks like setting the Super key to open the menu. That works great. Using Super key combos in lxqt-globalkeys works great. Unfortunately, lxqt-globalkeys can't handle window manager-specific actions unless exposed by a command or via `dbus`, which, unfortunately, Openbox does not. It could be considered an Openbox bug, but at least as far as I can tell, Openbox is the canonical window manager of LXQt.

##### System Information
* Distribution & Version: Lubuntu 18.10
* Kernel: 4.18.0-10-generic 
* Qt Version: libqt5core5a 5.11.1+dfsg-7ubuntu1
* liblxqt Version: liblxqt0 0.13.0-0ubuntu1
* Package version: lxqt-globalkeys 0.13.0-0ubuntu2

[1]: https://github.com/lxqt/lxqt/issues/1387
[2]: https://github.com/lxqt/lxqt/issues/1259#issuecomment-295770457
[3]: https://github.com/lxqt/lxqt/issues/331#issuecomment-184651925
[4]: https://github.com/lxqt/lxqt/issues/93

**Comment by jory18** [comment]
Same prolem here.

Version:Lubuntu 18.10

**Comment by Steedalion** [comment]
> Same prolem here.
> 
> Version:Lubuntu 18.10

Hi @jory18 ,
We (Lubuntu 18.10) are looking into the matter. A short term solution is disabling the menu shortcut (Super key) in lxqt shortcuts. You can follow the Lubuntu discussion [here](https://phab.lubuntu.me/T154).

**Comment by agaida** [comment]
I guess we can close this here - The super key is handled like a normal key - one could test it with `xev` - and i don't think that we re-invent the wheel and keyboard handling. So i would suggest to leave super key combinations to the window manager and not to handle the super key in lxqt-globalkeys.

Hint: Run `xev` from a terminal and keep the keys in charge pressed - compare the behaviour of 'shift' 'alt' 'ctrl' and 'super key' - have fun. 

@tsujan @palinek @yan12125 @luis-pereira - what do you think

**Comment by tsujan** [comment]
I think it can be closed.

> Openbox is the canonical window manager of LXQt.

Even if that's said in some LXQt doc (is it?), it's wrong (and should be removed from that doc). LXQt doesn't have a special relationship with Openbox.

**Comment by emr-rgb** [comment]
editing this post, no need to duplicate:

For anyone landing here from search: I posted a working fix for this exact
problem (Super key alone opens the LXQt main menu without conflicting with
WM/Openbox shortcuts) over on lxqt-globalkeys#259, since that's the more
relevant/active thread for it:

https://github.com/lxqt/lxqt-globalkeys/issues/259

Direct link to the full writeup + code (gist):
https://gist.github.com/emr-rgb/8e8c843ca32cc0b4427a86a9733d3202

Short version: don't grab the Super key at all — passively observe key
press/release via the X11 RECORD extension (no conflicts possible, since
RECORD doesn't grab), and trigger the menu directly over D-Bus when a lone
tap is detected. Full explanation and code at the links above.
